frontend/src/components/Psychologist/Profile/ConventionForm.jsx

```diff
@@ -56,6 +59,14 @@ const ConventionForm = ({ currentConvention, onConventionUpdated, checkDefaultVa
               ? universities.map(university => ({ value: university.id, label: university.name }))
               : []}
           />
+          {(convention.universityId === laSorbonneParisNord || convention.universityId === laSorbonne) && (
+            <Alert
+              description="Pensez à vérifier que l'université sélectionnée est exacte.
+            Dans le cas de La Sorbonne (75) et Sorbonne Paris Nord (93)."
```

Comment:
je preferais le wording de Sandrine mais a checker ?
```suggestion
            Il y a La Sorbonne (75) et Sorbonne Paris Nord (93)."
```

Reply:
Justement Sandrine a conseillé de prendre celui de Lina.